MissionHotpointTask hotpoint_task
---
bool result
